Makefile

```diff
@@ -78,11 +78,7 @@ out/Makefile: common.gypi deps/uv/uv.gyp deps/http_parser/http_parser.gyp \
 	$(PYTHON) tools/gyp_node.py -f make
 
 config.gypi: configure
-	if [ -f $@ ]; then
-		$(error Stale $@, please re-run ./configure)
-	else
-		$(error No $@, please run ./configure first)
-	fi
+	$(error Missing or stale $@, please re-run ./$<)
```

Comment:
Tiniest of nits: maybe s/re-run/run/ - in the 'missing' case, you probably haven't run configure before.